lib/fabrik/BaseManager.js

```diff
@@ -55,6 +55,14 @@ class BaseManager {
     );
   }
 
+  getTenantGuid(context) {
+    if (context.platform === 'cloudfoundry') {
+      return context.space_guid;
+    } else if (context.platform === 'kubernetes') {
```

Comment:
You can put 'cloudfoundry' and  'kubernetes' in constants as might be reused in other place of code base.

Reply:
👍

Reply:
Done